relay/sources/rudder-pkg/lib/rudder-pkg/rudderPkg.py

```diff
@@ -350,20 +350,21 @@ def upgrade_all(mode):
         currentVersion = rpkg.PluginVersion(utils.DB["plugins"][p]["version"])
         pkgs = plugin.Plugin(p)
         pkgs.getAvailablePackages()
+        latestversion = None
         if mode == "nightly":
             latest_packages = pkgs.getLatestCompatibleNightly()
             if (latest_packages is None):
-                print("No newer %s compatible versions found for the plugin %s"%(mode, p))
+                logging.error("No newer %s compatible versions found for the plugin %s"%(mode, p))
```

Comment:
I don't think that's an error but an info and maybe we can get rid of that message here because it will be also printed at the end by a logging.info

Reply:
I think we should only do the else par (with is not None in if)